src/cmd-buildextend-metal

```diff
@@ -267,8 +267,7 @@ cloud-image-size: "${cloud_image_size_mb}"
 EOF
 yaml2json image-dynamic.yaml image-dynamic.json
 cat "${image_json}" image-dynamic.json | jq -s add > image-for-disk.json
-platforms_json="${workdir}/tmp/platforms.json"
-yaml2json "${configdir}/platforms.yaml" "${platforms_json}"
+platforms_json="/usr/share/coreos-assembler/platforms.json"
```

Comment:
The way we are working with 'platforms.json' may change  as we progress with the OSBuild work. However, we don't have a clear vision about how it will be in the future yet, as we also have some similar bits under the osbuild composer to handle the cloud parameters.  We probably will know better when we start working with the cloud artifacts.

Reply:
@cgwalters definitely. We had a session with @ravanelli @jlebon and myself to discuss. We realize that what we're doing in this PR is not part of the longer term strategy.